aten/src/ATen/native/native_functions.yaml

```diff
@@ -104,6 +104,19 @@
 - func: _unpack_dual(Tensor(a) dual, int level) -> (Tensor(a) primal, Tensor tangent)
   variants: function
 
+# NOTE [Two New-zero Funcitons]
+# The following two functions are used internally for forward AD
+# We need two functions here because we'd like a version to use for non-batched
+# forward that can accept non-strided tensors, and another variant specifically
+# when self is batched. Note that this is possible because a batching rule does
+# not need to redispatch to the same function.
+- func: _new_zeros_with_same_meta(Tensor self, Tensor other) -> Tensor
+  variants: function
+  dispatch:
+    CompositeExplicitAutograd: _new_zeros_with_same_meta
```

Comment:
Why is this one explicit autograd?
The vmap key is above autograd, so you can override it even if it is implicit autograd.

Reply:
If it actually needs to be explicit, then we need a formula for it to specify the output differentiability.

Reply:
Maybe I should add a comment explaining that it only applies to functorch, in which case if we have `vmap(jvp` and the key orders are switched.

But per richard's comment offline, the current configuration does not work for the `vmap(vmap(jvp` case.

Reply:
Ok, I guess we want a comment and an entry in derivatives.yaml